dashboard/app/assets/config/manifest.js

```diff
@@ -0,0 +1,16 @@
+// Match the functionality of pre-manifest.js-controlled Sprokets; ie, include
+// only the application.{css.js} files and images, but not the other files in
+// the javascripts and stylesheets directories.
+// See https://github.com/rails/sprockets/blob/main/UPGRADING.md#manifestjs
+//
+//= link application.css
+//= link application.js
+//= link_tree ../images
+
+//= link_tree ../../../public/blockly/js
```

Comment:
note that we don't specify only javascript files here, just as the `config.assets.precompile` method doesn't; this ensures that the images in that directory are also loaded